### DISCUSSION TITLE
Package Manger(s) support

### DISCUSSION THREAD

**Discussion by Daniele-rolli** [Daniele-rolli/Beaver-Notes#8] [Package Manager]
This conversation has been started with the intent to not only share the efforts made in making Beaver Notes more accessible to the community but also to  engage the community in the process of making Beaver Notes available across various platforms. We are starting with apt as the first package manager to receive official support from the Beaver Notes project.

Road Map of package managers:

### Linux
- [x] Apt
- [x] Aur
- [x] Chaotic-AUR
- [x] Rpm
- [x] Flatpak 
### macOs
- [x] HomeBrew
### Windows
- [x] Scoop

To install Beaver Notes, visit the  [available package managers.](https://beavernotes.com/#/Packages)

**Comment by trymeouteh** [comment]
Please release app on...

**Linux**

Flatpak
Nix Package Manager

**Windows**
Scoop


And please continue releasing it as an AppImage, EXE and DMG.

Flatpak, Nix and Scoop are open source and decentralized package managers.

**Reply by Daniele-rolli** [reply]
Thank you once again for your feedback and your dedication to this project. I'll look into both Scoop and Nix when I have the opportunity, although they may not be at the top of the priority list. Rest assured, support for them will eventually be included.

Flatpak-wise, I can't make any promises as it falls outside my responsibilities to implement Flatpak support in the Electron builder. However, I will look into the possibility and aim to release a Flatpak version as soon as support becomes available. Thanks again for understanding.
